frontend/lib/ui/Modal/Modal.stories.tsx

```diff
@@ -5,22 +5,20 @@ import { Button } from "@kiesraad/ui";
 import { Modal } from "./Modal";
 
 export const DefaultModal: Story = () => (
-  <>
-    <Modal onClose={() => {}}>
-      <h2>Wat wil je doen met je invoer?</h2>
-      <p>
-        Ga je op een later moment verder met het invoeren van dit stembureau? Dan kan je de invoer die je al hebt gedaan
-        bewaren.
-        <br />
-        <br />
-        Twijfel je? Overleg dan met de coördinator.
-      </p>
-      <nav>
-        <Button size="lg">Invoer bewaren</Button>
-        <Button size="lg" variant="secondary">
-          Niet bewaren
-        </Button>
-      </nav>
-    </Modal>
-  </>
+  <Modal id="modal-title" onClose={() => {}}>
```

Comment:
I think that the Modal prop name 'id' is a bit confusing, at first I thought that there are now two identical id's on the page. Maybe rename it with a more descriptive name like `focusId` ?

Reply:
Added the title as a prop to modal, to be in line with the form and to not have to add this id as a prop in 8dc28d65183b3e99fae58abe59646b139e4ae842

Reply:
Ah even better!